import-export-cli/cmd/exportApps.go

```diff
@@ -0,0 +1,103 @@
+/*
+*  Copyright (c) WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+*
+*  WSO2 Inc. licenses this file to you under the Apache License,
+*  Version 2.0 (the "License"); you may not use this file except
+*  in compliance with the License.
+*  You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing,
+* software distributed under the License is distributed on an
+* "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+* KIND, either express or implied.  See the License for the
+* specific language governing permissions and limitations
+* under the License.
+ */
+
+package cmd
+
+import (
+	"fmt"
+	"path/filepath"
+
+	"github.com/wso2/product-apim-tooling/import-export-cli/impl"
+
+	"github.com/spf13/cobra"
+	"github.com/wso2/product-apim-tooling/import-export-cli/credentials"
+	"github.com/wso2/product-apim-tooling/import-export-cli/utils"
+)
+
+var exportAppsWithKeys bool
+var exportAppsFormat string
+var startFromBeginningForApps bool
+var isProcessCompletedForApps bool
+
+// ExportApps command related usage info
+const ExportAppsCmdLiteral = "apps"
+const exportAppsCmdShortDesc = "Export Apps for migration"
```

Comment:
Shall we change the description like so? or do we need the `for migration` part as well? does this new command only be used in a migration scenario?

```suggestion
const exportAppsCmdShortDesc = "Export Applications"
```

Reply:
Here the word migration is used in terms of moving resources from one environment to another. But +1 to make it simpler.